scripts/generateAssetData/generateRelatedAssetIndex.ts

```diff
@@ -9,8 +9,10 @@ const main = async () => {
     console.info(
       "Done. Don't forget to add a migration to clear assets state so the new assets are loaded.",
     )
+    process.exit(0)
```

Comment:
This was not a big issue locally as we could CMD + C it, but much bigger issue in proper CLI use.